paddle/fluid/operators/mkldnn/conv_mkldnn_op.cc

```diff
@@ -587,18 +583,9 @@ class ConvMKLDNNHandlerT
       post_operations.append_sum(sum_scale);
     }
 
-    if (fuse_activation == "hard_sigmoid") {
-      post_operations.append_eltwise(activation_scale,
-                                     dnnl::algorithm::eltwise_linear,
-                                     fuse_alpha,
-                                     fuse_beta);
-      post_operations.append_eltwise(
-          activation_scale, dnnl::algorithm::eltwise_clip, 0.0f, 1.0f);
-    } else if (fuse_activation != "") {
-      const auto activation_algorithm =
-          platform::AcquireActivationAlgorithm(fuse_activation);
-      post_operations.append_eltwise(
-          activation_scale, activation_algorithm, fuse_alpha, fuse_beta);
+    if (!ctx.Attr<std::string>("fuse_activation").empty()) {
```

Comment:
`!` is easy to miss, lets make a helper variable
```suggestion
    auto is_fuse_activation_not_empty = !ctx.Attr<std::string>("fuse_activation").empty();
    if (is_fuse_activation_not_empty) {
```